next.config.js

```diff
@@ -113,6 +120,9 @@ module.exports = {
   env: {
     NEXT_PUBLIC_VERSION: version,
   },
-
+  productionBrowserSourceMaps: true,
```

Comment:
optional: thought it'd be helpful in the future, although front-end isnt nearly as hard to debug